Doc/library/dialog.rst

```diff
@@ -73,7 +73,7 @@ listed below:
 
  | *defaultextension* - default extension to append to file (save dialogs)
 
- | *multiple* - when True, selection of multiple items is allowed
+ | *multiple* - when true, selection of multiple items is allowed
```

Comment:
I'm not entirely sure what this change accomplishes. Could this one not be ``True`` as well?

```suggestion
 | *multiple* - when ``True``, selection of multiple items is allowed
```

Reply:
No, this should be "true". Passing `multiple=1` or other true value has the same effect as passing `multiple=True`.

Reply:
@serhiy-storchaka 
> Passing multiple=1 or other true value has the same effect as passing multiple=True.

Personally, I see the constant ``True`` and a true value as being equivalent for the purposes of describing arguments in the documentation. Is there a case where the distinction is significant for passing arguments?

Reply:
The change is correct.  'x is true' means 'bool(x) is True' (both as English statement and Python expression)', which is a looser condition than 'x is True' (as a Python expression).

Reply:
> 'x is true' means 'bool(x) is True' (both as English statement and Python expression)', which is a looser condition than 'x is True' (as a Python expression).

Hmm, I think I see what you mean. In that case, should we use "x is true" as opposed to "x is True" in any case when describing the input to arguments?

I can't think of any case in CPython where we specifically require the passing of a boolean constant rather than a boolean value. So, the "looser condition" pretty much always applies in those cases.